### DISCUSSION TITLE
Welcome to Spring AI Discussions!

### DISCUSSION THREAD

**Discussion by markpollack** [spring-projects/spring-ai#34] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by madhumeher0037** [comment]
hello @markpollack 

Im working on integrating spring AI into my project.
Do you know which springboot version supports the Spring AI.My project is currently running springboot <version>2.4.2</version>.
